includes/REST_API/Stories_Controller.php

```diff
@@ -388,6 +389,12 @@ public function get_collection_params() {
 			'default'     => false,
 		];
 
+		$query_params['web_stories_demo'] = [
+			'description' => __( 'Load demo data.', 'web-stories' ),
+			'type'        => 'string',
```

Comment:
Why not a bool?

Reply:
I used a string here, as it was a string before and it was a nice drop in replacement. Willing to change it, if it makes sense.

Reply:
It's a boolean flag after all, so we should use the correct type here

Reply:
If we can make it a bool then I a happy to approve 👍

Reply:
Changed in https://github.com/google/web-stories-wp/pull/5507/commits/a12720f45b99157d1675ef5bfdba28fed0ef4f0a